src/compas_view2/objects/pointobject.py

```diff
@@ -25,12 +27,21 @@ def init(self):
         colors = [self.color or self.default_color]
         elements = [0]
         self._points = {
-            'positions': make_vertex_buffer(list(flatten(positions))),
+            'positions': make_vertex_buffer(list(flatten(positions)), True),
```

Comment:
what about all the other buffers? should we make them them all dynamic? will that impact performance?

Reply:
let me make some test

Reply:
tried on 1000 cubes, don't really see any performance change when setting dynamic default to true